.gitignore

```diff
@@ -18,6 +18,7 @@ npm-debug.log
 yarn-error.log
 local_version.txt
 .env
+.env.bak
```

Comment:
Are you sure this should be committed to the repo?

Reply:
Yes, this is to prevent me from accidentally checking in my personal backup file.

Reply:
```
git reset HEAD .env.bak
```

will unstage that file from git and it won't be added to git unless you explicitly add it.

Reply:
That would only help if I catch myself first. 

That said if you don't like this change I'm ok removing it, not a big deal.

Reply:
I've removed the change